boards/arm/nrf52840dk_nrf52840/doc/index.rst

```diff
@@ -192,18 +192,13 @@ more than one UART for connecting peripheral devices:
       };
 
       &uart1 {
-        compatible = "nordic,nrf-uarte";
-        current-speed = <115200>;
-        status = "okay";
-        pinctrl-0 = <&uart1_default>;
-        pinctrl-1 = <&uart1_sleep>;
-        pinctrl-names = "default", "sleep";
+        pinctrl-0 = <&uart1_default_alt>;
+        pinctrl-1 = <&uart1_sleep_alt>; /* /delete-property/ not defined */
```

Comment:
I don't quite understand what that comment here is supposed to say. If you don't need the sleep state, use `/delete-property/ pinctrl-1`?

Reply:
oops, missed `if`

Reply:
I'm afraid that even with this "if" it can still be unclear to users.

Reply:
what about now?

Reply:
much better :)